test/Microsoft.DotNet.Docker.Tests/ImageTests.cs

```diff
@@ -37,7 +37,27 @@ public static IEnumerable<object[]> GetVerifyImagesData()
                 testData.AddRange(new List<ImageDescriptor>
                     {
                         new ImageDescriptor {DotNetCoreVersion = "2.0", OsVariant = "jessie"},
-                        new ImageDescriptor {DotNetCoreVersion = "2.1", RuntimeDepsVersion = "2.0", OsVariant = "jessie"},
+                        new ImageDescriptor
+                        {
+                            DotNetCoreVersion = "2.0",
+                            OsVariant = "stretch",
```

Comment:
<nit> I think it would be valuable to put SdkOsVariant after OsVariant.

Reply:
Clarification - `valuable` in terms of readability?

Reply:
Grouping similar parameters adds value in that it helps call them out and that they don't get overlooked.  If you are browsing quickly and see the OsVariant you may overlook that there is also an SdkOsVariant.

Reply:
Thanks.